index.js

```diff
@@ -1,3 +1,51 @@
+const isExist = (ref)       => typeof ref != 'undefined' && ref!=null;
+const isArray = (ref)       => typeof ref=='object' && ref!=null;
+const isFunction = (ref)    => typeof ref=='function' && ref!=null;
+const isObject = (ref)      => typeof ref=='object' && ref!=null;
+const getValue = (ref, def) => isExist(ref) ? ref : def;
+
+function cloneObject(obj)
+	{
```

Comment:
brackets here are used differently than in other code

- [ ] either use same notation as is used in surrounding code or rewrite surrounding code to use your annotation